test/test_parallel_executor.cpp

```diff
@@ -194,18 +140,16 @@ TEST(ParallelExecutor, ExecuteOnSingleThread) {
 
   std::vector<Tensor> tf_input_tensors{x, y};
   shared_ptr<ngraph::runtime::Executable> ng_exec;
-
+  shared_ptr<PipelinedTensorsStore> pts;
```

Comment:
We dont have any tests for this PR specifically (integration of cache). Can we run RESNET50 and check there is no cache miss. If we can create a test like this it will be even better. Suggestions welcome.